### PR TITLE
Add support for WorkloadALTSConfig in google_container_cluster (Beta)

### PR DESCRIPTION
**Pull request by AshZhang**
Fixes https://github.com/hashicorp/terraform-provider-google/issues/16793.

[WorkloadALTSConfig](http://cloud/kubernetes-engine/docs/reference/rest/v1beta1/projects.locations.clusters#Cluster.WorkloadALTSConfig) is a parameter in the beta GKE Cluster API used to configure authentication for Directpath connections. This PR adds support to the google-beta provider.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
container: added `workload_alts_config` field to `google_container_cluster` resource (beta)
```

### CODE REVIEW

File: mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb [hunk 1]
```diff
@@ -2072,6 +2072,24 @@ func ResourceContainerCluster() *schema.Resource {
 					},
 				},
 			},
+			<% unless version == 'ga' -%>
+				"workload_alts_config": {
+					Type:             schema.TypeList,
+					Optional:         true,
+					Computed:		  true,
```
Comment (slevenick) [inline mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb:2079]: Looks like there's some spacing issues here

File: mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.erb [hunk 2]
```diff
@@ -9259,3 +9299,81 @@ resource "google_container_cluster" "without_confidential_boot_disk" {
...
+			workload_pool = "${data.google_project.project.project_id}.svc.id.goog"
+		}
+		deletion_protection = false
+	}
+`, projectID, networkName, subnetworkName, name)
+}
+<% end -%>
+
+<% unless version == 'ga' -%>
+func testAccContainerCluster_withWorkloadALTSConfig_update(projectID, networkName, subnetworkName, clusterName string) string {
```
Comment (slevenick) [inline mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.erb:9343]: You could replace this second config by just passing the boolean value of enable_alts as a parameter

**Review by slevenick — CHANGES_REQUESTED** [review]
LGTM overall, just some small changes requested

File: mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb (hunk 1 above)
Comment (AshZhang) [reply mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb]: Fixed.

File: mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.erb (hunk 2 above)
Comment (AshZhang) [reply mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.erb]: Done, removed the extra config and added a boolean parameter to the original config.

**Review by slevenick — APPROVED** [review]

**Pull request merged**